data-raw/R2R_HRL_habitat_inputs/add_SBR_hrl_data.Rmd

```diff
@@ -0,0 +1,815 @@
+---
+title: "Add HRL Habitat Data"
+author: "Liz Stebbins (FlowWest)"
+date: "`r Sys.Date()`"
+output: html_document
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>", 
+  fig.dim = c(8, 6)
+)
+```
+
+```{r setup, include = FALSE}
+library(DSMhabitat)
+library(tidyverse)
+library(lubridate)
+library(DSMflow)
+library(readxl)
+library(sf)
+#remotes::install_github("flowwest/riceflows4ff")
+library(riceflows4ff)
+library(scales)
+```
+
+# Overview
+
+As part of a R2R scenario, habitat commitments made under the Healthy Rivers and Landscapes (HRL) project were included for the following watersheds:
+
+-   American River
+-   Feather River
+-   Mokelumne River
+-   Yuba River
+-   Tuolumne River
+
+Staff from the Tuolumne River and Yuba River provided data directly for modeling anticipated flow-to-suitable area relationships under the HRL commitments. These streams developed methods to scale an existing habitat flow-to-suitable area relationship based on existing and proposed habitat projects and can be read about in more detail *here* (TODO).
+
+The Mokelumne, American, and Feather Rivers did not provided updated relationships, and for consistency across the basin R2R used the flow-to-suitable area relationships provided for the [Scientific Basis Report (SBR), Section 6](https://www.waterboards.ca.gov/waterrights/water_issues/programs/bay_delta/docs/2023/staff-report/app-g2.pdf).
+
+# Methods
+
+For use in the R2R scenario, we implemented the following methods:
+
+1)  Read in the flow-to-suitable area relationship used in the SBR
```

Comment:
For all of these that have inchannel rearing we will want to update to make sure we apply to both fry and juv rearing not just juv.